zulipterminal/ui_tools/views.py

```diff
@@ -665,10 +665,16 @@ def menu_view(self) -> Any:
         self.view.starred_button = StarredButton(self.controller,
                                                  width=self.width)
 
+        self.view.mentioned_button = MentionedButton(
+            self.controller,
+            width=self.width,
+            count=self.model.unread_counts['all_mentions'])
+
         menu_btn_list = [
             self.view.home_button,
             self.view.pm_button,
             self.view.starred_button,
+            self.view.mentioned_button,
```

Comment:
Perhaps we should switch the order here, to match the webapp, at least for now?
(the order of the `MentionedButton` code relative to the `StarredButton` too, if so)

Reply:
Sure. I changed the order of the buttons and code in respective places.